src/DBFTPlugin/PrepareRequest.cs

```diff
@@ -30,7 +30,7 @@ public override void Deserialize(BinaryReader reader)
             Version = reader.ReadUInt32();
             PrevHash = reader.ReadSerializable<UInt256>();
             Timestamp = reader.ReadUInt64();
-            TransactionHashes = reader.ReadSerializableArray<UInt256>(Block.MaxTransactionsPerBlock);
+            TransactionHashes = reader.ReadSerializableArray<UInt256>((int)ProtocolSettings.Instance.MaxTransactionsPerBlock);
```

Comment:
there is no Instance now since https://github.com/neo-project/neo/pull/2336 is closed.

Reply:
Fixed.